client/components/resource-details/ResourceInfo.tsx

```diff
@@ -95,46 +102,32 @@ export const ResourceInfo = ({ resource }: ResourceInfoProps) => {
 
       <Description>{description}</Description>
       <Gallery links={galleryImages} />
-      <Reviews reviews={[]} />
+      <Reviews reviews={reviews} />
     </div>
   )
 }
 
-const Categories = ({ tags }: { tags: Array<string> }) => (
-  <CategoriesWrapper>
-    {tags.map((t, i) => (
-      <TagBubble key={`category-tag-${i}`} colour={colours.tagColours[t]}>
-        {t}
-      </TagBubble>
-    ))}
-  </CategoriesWrapper>
-)
-
-const GalleryWrapper = styled.div`
-  width: 50%;
-`
-
-const Gallery = ({ links }: { links: Array<string> }) => {
+const Gallery = styled(({ className, links }: { links: Array<string>; className?: string }) => {
   const images = links.map((link) => {
     return {
       original: link,
       thumbnail: 'https://picsum.photos/id/1018/250/150/',
     }
   })
-
   return (
-    <GalleryWrapper>
+    <div className={className}>
```

Comment:
IIRC this is a common React idiom for passing in styles, but we are using `styled-components` so I'm not sure we actually need it